user.css

```diff
@@ -1619,9 +1637,9 @@ option {
 }
 
 .rVxzkDirgkuRPv5V1HYF[aria-hidden="true"],
-.zcsxy3fvYCmZGcE7VzW7[aria-hidden="true"] {
+.main-coverSlotCollapsed-container[aria-hidden="true"] {
   left: calc(100vw - 178px);
-  top: -245px;
+  top: -244px;
```

Comment:
Seems very unnecessary to change a 1 pixel difference as it is often subjective, better leave it as-is